14/umbraco-workflow/getting-started/workflow-content-app.md

```diff
@@ -0,0 +1,99 @@
+
+## Configuration
+
+The Configuration sub-section provides an interface for configuring the content approval flow for the current node. It also displays any Inherited or Document type approval flows applied to the current content node.
+
+In multi-lingual sites, each variant can have its own approval flow. By default, new variants inherit the configuration set on the default language.
+
+For example, German variants can be approved by the German speakers group, while English variants are approved by the English speakers group.
+
+![Configuration sub-section](images/Configuration-sub-section-v13.png)
+
+### Content Approval Flow
+
+You can add different groups for different stages of content approval flow. Content Approval flow groups can be reordered via drag and drop. You can also apply the approval flow either for publish and unpublish workflow or only publish workflow.
+
+![Content approval flow](../../../10/umbraco-workflow/getting-started/images/content-approval-flow.gif)
+
+#### Approval Flow Types
+
+Approval Flows are available in three types: Content approval flow, Inherited approval flow, and Document type approval flow.
+
+A given content node may have all three approval flow types applied but only one will be applied as per the following order of priority:
+
+* **Content approval flow:** set directly on a content node via the **Configuration** section in the **Workflow** tab . This type will take priority over all others.
+* **Document type approval flow:** set in the **Settings** section. This approval flow will apply to all content nodes of the selected Document Type unless the node has a Content approval flow set. This feature requires a license.
+* **Inherited approval flow:** if a node has no Content approval flow set, nor a flow applied to its Document Type, Umbraco Workflow will traverse the content tree until it finds a node with a Content approval flow and will use this flow for the current change.
+
+![Approval Flow Types](../../../10/umbraco-workflow/getting-started/images/approval-flow-types.png)
+
+Review the current responsibilities for Approval Groups in the **Roles** tab of the **Approval Groups** section for **Node-based approvals** and **Document type approvals** only. For more information see the [Roles](../workflow-section/approval-groups.md#roles) section in the [Approval Groups](../workflow-section/approval-groups.md) article.
+
+![Approval Groups Roles](../../../10/umbraco-workflow/getting-started/images/approval-groups-roles.png)
+
+Document type approval flows can also include conditional stages i.e., only include **Translators** in the workflow when the **Description** property has changed. For more information on settings conditions in Document type approval flows, see the [Document type approval flows](../workflow-section/workflow-settings.md#document-type-approval-flows) section in the [Workflow Settings](../workflow-section/workflow-settings.md) article.
```

Comment:
Can this first sentence be written without the use of "i.e."?
It's on [the list of Terms](https://github.com/umbraco/UmbracoDocs/blob/main/.github/styles/UmbracoDocs/Terms.yml) that we want to avoid using 💪